resources/views/pages/home.blade.php

```diff
@@ -29,27 +29,18 @@
             <img src="{{asset('images/angle-right.svg')}}" alt="" class="arrow-right2">
           </div>
           <div class="gallery">
-            <div class="card1 card">
+          @foreach($recent_projects as $recent_project)
+            <div class="card{{$loop->iteration}} card">
                 {{-- <p class="tag">New</p> --}}
               <div class="project">
-                  <p>Contruction of Lagos-Ibadan Express road <br>
-                    Ministry of Power, Works and Housing</p>
+                  <p>{{$recent_project->project}}<br></p>
                   <div class="d-flex justify-content-between mt-4 align-items-center">
                       <p>Cost of Project: </p>
-                      <p id="cost">&#8358;20bn</p>
+                      <p id="cost">{{$recent_project->amount_spent}}</p>
```

Comment:
This should be converted to proper money format with the Naira currency prepended. I believe there is an `amount()` helper method on `App\Expense.php`